configure.ac

```diff
@@ -0,0 +1,116 @@
+# Configure template for Gerbil system.
+
+# Copyright (c) 2019 by drewc <me@drewc.ca>, All Rights Reserved.
+
+# Process this file with autoconf to produce a configure script.
+
+AC_INIT(Gerbil,v0.15.1,me@drewc.ca,gerbil-v0_15_1)
+PACKAGE_SHORTNAME="gerbil"
+
+AC_SUBST(PACKAGE_SHORTNAME)
+AC_SUBST(PACKAGE_NAME)
+AC_SUBST(PACKAGE_VERSION)
+AC_SUBST(PACKAGE_STRING)
+AC_SUBST(PACKAGE_BUGREPORT)
+AC_SUBST(PACKAGE_TARNAME)
+
+AC_COPYRIGHT([[Copyright (c) 2019 by drewc <me@drewc.ca>, All Rights Reserved.]])
+
+
+# Libxml
+AC_ARG_ENABLE(libxml,
+AC_HELP_STRING([--enable-libxml],
+[build std/xml libraries - requires libxml2 (default is NO)]),
+ENABLE_LIBXML=$enableval,
+ENABLE_LIBXML=no)
+
+if test "$ENABLE_LIBXML" = yes; then
+sed -i -e 's|enable libxml #f|enable libxml #t|g' src/std/build-features.ss || true
+LIBS="$LIBS -lxml2"
```

Comment:
do we actually need to add the libraries to `$LIBS`? The build script adds ld dependencies as needed.

Reply:
Some are needed when linking, some are not. I can't quite remember why, but I think it has to do with timing and if the link is used at compile time.